modules/core/src/main/kotlin/datamaintain/core/step/Scanner.kt

```diff
@@ -13,16 +14,20 @@ class Scanner(private val context: Context) {
 
         return rootFolder.walk()
                 .filter { it.isFile }
-                .map {
-                    val tags = mutableSetOf<Tag>()
+                .map { FileScript(it.toPath(), context.config.identifierRegex,
+                        buildTags(context.config, rootFolder, it).toSet()) }
+                .sortedBy { it.name }.toList()
+    }
 
-                    if (context.config.doesCreateTagsFromFolder) {
-                        tags.addAll(buildTagsFromFolder(rootFolder, it));
-                    }
+    private fun buildTags(config: DatamaintainConfig, rootFolder: File, file: File): Set<Tag> {
+        val tags = mutableSetOf<Tag>()
 
-                    FileScript(it.toPath(), context.config.identifierRegex, tags.toSet())
-                }
-                .sortedBy { it.name }.toList()
+        if(config.doesCreateTagsFromFolder)
+            tags.addAll(buildTagsFromFolder(rootFolder, file))
```

Comment:
I like space between `if` and `(`. I prefer to have clearly identified block `if` using `{` and `}` event if this block contains only one line.

Reply:
Alright, I've added curly brackets in commit e7b6a570a7e5f7d0d26a7a70c782eb647eb358c9